pkgs/development/libraries/botan/3.0.nix

```diff
@@ -0,0 +1,9 @@
+{ callPackage, fetchpatch, lib, ... } @ args:
+
+callPackage ./generic.nix (args // {
+  baseVersion = "3.1";
+  revision = "1";
+  sha256 = "sha256-MMhP6RmTapj+9TMfJGxiqiwOTSCFstRREgf2ogr6Oms=";
+  # reconsider removing this platform marking, when MacOS uses Clang 14.0+ by default.
+  meta.badPlatforms = [ lib.platforms.darwin ];
```

Comment:
Ahh you would need to `overrrideAttrs` this derivation and add meta.badPlatforms there. My bad. Or you could add a `batPlatforms` flag to generic.nix

Reply:
I added a badPlatforms Flag.